README.md

```diff
@@ -30,28 +30,28 @@ RELEASE_FEED_TOKEN=aaaabbbbccccddddeeeeffffgggg0000
 Second, install dependencies:
 
 ```bash
-$ composer install
+composer install
```

Comment:
This does not match the deactivated rule number 14. This also applies to all other CLI commands.

https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/md014.md